pkg/application/app.go

```diff
@@ -15,6 +16,10 @@ import (
 	"github.com/giantswarm/clustertest/pkg/organization"
 )
 
+const (
+	defaultValuesContents = "\n"
```

Comment:
This feels a bit awkward, don't you think the consumer should be able to handle the default value of an empty string?